downloader/src/main/kotlin/vcs/MercurialWorkingTree.kt

```diff
@@ -25,7 +25,7 @@ import org.ossreviewtoolkit.downloader.WorkingTree
 import org.ossreviewtoolkit.model.VcsType
 import org.ossreviewtoolkit.utils.common.ProcessCapture
 
-class MercurialWorkingTree(workingDir: File, vcsType: VcsType) : WorkingTree(workingDir, vcsType) {
+internal class MercurialWorkingTree(workingDir: File, vcsType: VcsType) : WorkingTree(workingDir, vcsType) {
```

Comment:
Should the commit title indicate that it's a "breaking" change?

Reply:
Strictly speaking yes, otherwise @nnobelis will complain again 😉 I'll change it.

Reply:
Ok, wow, I just realized that GitHub now keep your approval even thou I did not only rebase, but also reword a commit message. So GitHub realizes that the actual code in the PR did not change, this is nice!